packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -166,7 +166,13 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
      * @param error - optional error object to log
      */
     public sendErrorEvent(event: ITelemetryErrorEvent, error?: any) {
-        this.sendTelemetryEventCore({ ...event, category: "error" }, error);
+        this.sendTelemetryEventCore({
+            // ensure the error field has some value,
+            // this can and will be overridden by event, or error
+            error: event.eventName,
```

Comment:
I'm not sure what benefits this change brings.
More content to push through the stack / pay for.

Reply:
Not sure how to think about telemetry cost, honestly. Would be interesting to get data from Loop org as a frame of reference.

Reply:
the volume of errors is low intentionally, and most have an error field. this will also be the un-namespaced event name, so it'll be short. i really don't see a problem here. we add much for verbose logs without impact analysis

Reply:
the benefit is really quality of life improvement, it allows me to keep my queries simpler